Source/WebCore/testing/Internals.h

```diff
@@ -451,6 +451,7 @@ class Internals final
     bool hasCorrectionIndicatorMarker(int from, int length);
 #if ENABLE(WRITING_TOOLS)
     bool hasWritingToolsTextSuggestionMarker(int from, int length);
+    bool hasTransparentContentMarker(int from, int length);
```

Comment:
`TransparentContent` exists independent of `ENABLE(WRITING_TOOLS)`, so I wouldn't put it behind the guard.